### PR TITLE
CODENVY-424: Configure Che Vagrant to setup Proxy

### PR DESCRIPTION
**Pull request by mmorhun**
Signed-off-by: Mykola Morhun <mmorhun@codenvy.com>

### CODE REVIEW

**Comment by mmorhun** [comment]
@garagatyi take a look please

File: Vagrantfile [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Set to "<proto>://<user>:<pass>@<host>:<port>"
+$http_proxy  = "http://testuser:testpassword@proxy.ua.codenvy-dev.com:3128"
```
Comment (garagatyi) [inline Vagrantfile]: I think it should not be set by default

**Comment by garagatyi** [comment]
@eivantsov @riuvshin @skryzhny please take a look

**Comment by garagatyi** [comment]
Since I have no experience with vagrant files, up to you @mmorhun

**Comment by riuvshin** [comment]
+1

**Pull request merged**